iceoryx_hoofs/test/moduletests/test_cxx_string.cpp

```diff
@@ -2348,4 +2348,160 @@ TEST(String100, FindLastOfForNotIncludedSTDStringFails)
     res = testString.find_last_of(testStdString, 50U);
     EXPECT_THAT(res.has_value(), Eq(false));
 }
+
+/// @note constexpr char& at(const uint64_t pos) noexcept
```

Comment:
If you use
```cpp
// BEGIN tests for at()
TYPED_TEST(stringTyped_test, Blup)
{
    //...
}
// END tests for at()
```

There would even be code folding available on some IDEs, maybe even vim supports this